internal/providers/service.go

```diff
@@ -58,6 +58,9 @@ type ProviderService interface {
 	DeleteProvider(ctx context.Context, provider *db.Provider) error
 }
 
+// TypeGitHubOrganization is the type returned from the GitHub API when the owner is an organization
+const TypeGitHubOrganization = "Organization"
```

Comment:
Nit: doesn't have to be exported
(not needed in this PR even if you agree the const doesn't have to be exported)